docs/dev/api_integration.md

```diff
@@ -0,0 +1,212 @@
+
+The images must meet the following requirements:
+
+| Image Requirements | Description                                                                                                                                       |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Type               | `.jpg` or `.png`.                                                                                                                                 |
+| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
+| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
+| Name               | Use letters, numbers, underscores, and hyphens. Do not use spaces.                                                                           |
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the media carousel which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog API Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog API Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][21].
+
+## Open a pull request
+
+Open a pull request in the [`integrations-extras` repository][10] that adds images (such as logos and images) and asset files (such as `Changelog.md`, `README.md`, and `manifest.json`) to your API integration's tile-only listing in the [Integrations page][11]. Automatic tests run checks to verify that your pull request is in good shape and contains all the required content to be updated.
+
+## Review process
+
+Once your pull request passes all the checks, reviewers from the `Datadog/agent-integrations` and `Datadog/documentation` teams provide suggestions and feedback on best practices.
```

Comment:
```suggestion
Once your pull request passes all the checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace` and `Datadog/documentation` teams provide suggestions and feedback on best practices.
```
Not sure if that label actually gets added but someone from our team reviews the PR as well.